recipes/libpq/all/conanfile.py

```diff
@@ -127,6 +127,8 @@ def build(self):
                         if not self.options.shared:
                             self.run("perl build.pl libpgport")
         else:
+            if self.settings.os == "Windows":
+                tools.replace_in_file(os.path.join(self._source_subfolder, "configure"), 'LIBS="-lz ', 'LIBS="')
```

Comment:
Would this also work?
```suggestion
                tools.replace_in_file(os.path.join(self._source_subfolder, "configure"), 'LIBS="-lzlib ', 'LIBS="')
```

Reply:
`LIBS="-lzlib ` doesn't exist, how could it work?